zeroex/order.go

```diff
@@ -351,17 +352,17 @@ func (s SignedOrder) MarshalJSON() ([]byte, error) {
 	}
 
 	signedOrderBytes, err := json.Marshal(SignedOrderJSON{
-		MakerAddress:          s.MakerAddress.Hex(),
+		MakerAddress:          strings.ToLower(s.MakerAddress.String()),
 		MakerAssetData:        makerAssetData,
-		MakerAssetAmount:      s.MakerAssetAmount.String(),
+		MakerAssetAmount:      strings.ToLower(s.MakerAssetAmount.String()),
```

Comment:
@hrharder the `MakerAssetAmount` should be a numerical number and therefore does not need to be lowercased.

Reply:
Ahh yes -- thank you 🤦‍♂